docs/content/virtualserver-and-virtualserverroute.md

```diff
@@ -0,0 +1,11 @@
+---
+title: VirtualServer and VirtualServerRoute Resources
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
+
+
+This doc is now available at https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/
```

Comment:
delete this doc and add an alias in the new file's frontmatter

Reply:
done